pages/associations.js

```diff
@@ -0,0 +1,47 @@
+import Head from 'next/head';
+import Header from '../components/header';
+import Panel from '../components/panel';
+import Information from '../components/information';
+import Value from '../components/value';
+import Easy from '../components/easy';
+import VirtualOffice from '../components/virtualOffice';
+import FAQ from '../components/faq';
+import Footer from '../components/footer';
+import Team from '../components/team';
+import GetEarlyAccessView from '../components/earlyAccessView';
+import TraditionView from '../components/traditionView';
+import SolutionView from '../components/solutionView';
+import { associationCopy } from '../utils/copyright_variables';
+
+export default function Home() {
```

Comment:
this is fine, but we should be only having to declare Home() once - following the DRY principles. 

For the sake of speed, let's move on. This will suffice

Reply:
You're right. Its an easy fix.